azure-servicebus/src/main/java/com/microsoft/azure/servicebus/ClientSettings.java

```diff
@@ -18,7 +18,12 @@
     private RetryPolicy retryPolicy;
     private Duration operationTimeout;
     private TransportType transportType;
-    
+
+    private String proxyHostName = null;
+    private int proxyHostPort = 0;
+    private String proxyUserName = null;
+    private String proxyPassword = null;
```

Comment:
For now. You can check this in. But Sreeram is also working on changing these setters/getters. I had suggested a change to the way proxy settings are accepted. When that change is made, we will make the same change here too.

Reply:
@binzywu mentioned those same changes - I can make them if you would like, it would not take long. Let me know!